packages/api2/src/periods/periods.controller.ts

```diff
@@ -62,7 +70,7 @@ export class PeriodsController {
   @ApiParam({ name: 'id', type: String })
   async findOne(
     @Param('id', ObjectIdPipe) id: Types.ObjectId,
-  ): Promise<Period> {
+  ): Promise<PeriodDetailsDto> {
```

Comment:
Check all period endpoints so that they return the correct DTO. In many cases `Period` is listed when it should be `PeriodDetailsDto`.